web/yo/app/scripts/controllers/gene.js

```diff
@@ -520,79 +542,87 @@ angular.module('oncokbApp')
                         }
                     }
                 }
-            }
-
+            };
+            $scope.acceptChangesByPerson = function(userName) {
+                if ($scope.status.isDesiredGene) {
```

Comment:
Nope. If it is not desired gene, we still need to accept changes and update front end models.